mock-core-configs/mock-core-configs.spec

```diff
@@ -1,6 +1,10 @@
+%if 0%{?epel} == 8
+%global python3 /usr/libexec/platform-python
```

Comment:
Really `%epel`, not `%rhel`?  Also, what if `%rhel == 9` or higher?

Reply:
Good catch, I was just thinking of the EPEL branch when putting this together as it's an epel only package. Slip of the mind.

Reply:
@praiskup Another alternative is to use `%el8` instead (thanks @carlwgeorge) as to prevent any issues with rebuild projects that may clear out `rhel` macro.

Reply:
Modified to `0%{?el8}`, this should be all set for merging unless there are any other areas that seem to pose a potential problem.